src/style.css

```diff
@@ -1,3 +1,83 @@
+:root {
+  --main-color: #3498db;
+  --header-height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding for your links instead